README.md

```diff
@@ -49,6 +49,7 @@ These sections show how to use the SDK to perform various authentication/authori
 11. [Tenant selection](#tenant-selection)
 12. [Logging Out](#logging-out)
 13. [History](#history)
+14. [My Tenants](#my-tenants)
```

Comment:
Don't we have one for Me? Let's have them together.

Reply:
We dont 
Placed next to history, which is similar, and is next to it in the code